palpatines-api/README.md

```diff
@@ -0,0 +1,132 @@
+        .
+        .
+    
+    "8521fac28eabd4b8b3c92141b58d9099:2a0913556f34bded73d0d8fb16e4912abb83ac9b147dacb404da6d3fd10e8ea3cdd4a23b3c84bbe0b3c3aef1981653a588ae24d22a8c7c12dc1a329182e30457bbc0190e36bb81f7eac3830b121d5360429220527581f51f7c2c9a79c408b030a34381df65cd2b52cb6e9d7f9d04286ec9db8f6c82a6356b093a1f07297cb025b980f2d10fdbf8af9e9501d4c912b56fdd11f374b7aa30cb98be3bdf13795efbdad2b4f311aad7a0cca7be55395b9aede1b5278294f859a23a3e771e26db4d41407859d9fbcbf599312589ae09eba7258f97eb7c88ccd2e05e97d9c78236a8a00b3c0c35c72d57279af6a192a745a0d9f859c4b1effa301d6b8efba3c48cdbd77c6fd92792e9bec2471918c86d0206a85383fde2eb38edfcd527c5989c813964a45b4200128ced578ca939cbeaff5c05f871e10fec0210629b8cd8dee8ef8242d717d8a4c87088066d137c98adf8bcd6b20edb68293285558eb0571ee3decbe4088a1414a6ef7d9c2963d6deb246bb56dfc81859478fad4c80e50888a3cc2cb07f8ae5e267c67f7984f153106904369eb01ba328fcb7289705aa45795df82e7b8287e53996c9063cd9266a4b128d0f810444a2ddee4be9ee83da5016fd98778f"
+]
+```
+
+
+Response:
+```
+[
+    {
+        "name": "Luke Skywalker",
+        "height": "172",
+        "mass": "77",
+        "hair_color": "blond",
+        "skin_color": "fair",
+        "eye_color": "blue",
+        "birth_year": "19BBY",
+        "gender": "male",
+        "homeworld": "https://swapi.co/api/planets/1/",
+        "films": [
+            "https://swapi.co/api/films/2/",
+            "https://swapi.co/api/films/6/",
+            "https://swapi.co/api/films/3/",
+            "https://swapi.co/api/films/1/",
+            "https://swapi.co/api/films/7/"
+        ],
+        "species": [
+            "https://swapi.co/api/species/1/"
+        ],
+        "vehicles": [
+            "https://swapi.co/api/vehicles/14/",
+            "https://swapi.co/api/vehicles/30/"
+        ],
+        "starships": [
+            "https://swapi.co/api/starships/12/",
+            "https://swapi.co/api/starships/22/"
+        ],
+        "created": "2014-12-09T13:50:51.644000Z",
+        "edited": "2014-12-20T21:17:56.891000Z",
+        "url": "https://swapi.co/api/people/1/"
+    }
+]
+```
+
+##### Quota
+
+You are limited to 50,000 requests per month and 25 requests per second. 
+
+#### SWAPI (Star Wars API)
+
+https://swapi.co/documentation 
+
+
+#### Submission Instructions
+
+* Use any packages or libraries which help you to complete these tasks
+* When you are finished, please provide us with a link to the completed repository (we use GithHub, but other services 
+are completely fine)
+* Please be sure to mention what version of Node you used
+* Describe any challenges that made the task more difficult
+Be prepared to discuss how you might improve your program to handle  a galaxy with millions or even billions of unique 
```

Comment:
Change 

Describe any challenges that made the task more difficult Be prepared to discuss how you might improve your program to handle  a galaxy with millions or even billions of unique citizens and homeworlds

to 

Describe any challenges that made the task more difficult and how you might improve your program to handle a galaxy with millions of unique citizens and homeworlds. Be prepared to discuss this in the technical interview